src/base/OpaqueId.hh

```diff
@@ -26,7 +26,7 @@ namespace celeritas
  * This allows type-safe, read-only indexing/access for a class. The value is
  * 'true' if it's assigned, 'false' if invalid.
  */
-template<class Instantiator, class T = std::size_t>
+template<class Instantiator, class T = unsigned int>
```

Comment:
Should we explain (at least in the commit log) the advantage of the switch from `size_t` to `unsigned int`?

Reply:
Oops, yes we should -- that got lumped in with the other commits I did. 😬

Reply:
I have extended the comment in the git log.